### PR TITLE
feat(value-expression-plugin): Allows load/save of preset data

### LINKED ISSUE #279: Plugin: Value Expression should load/save assignments from/to presets
The title.

### PR DESCRIPTION
**Pull request by 2xAA**
Fixes #279

### CODE REVIEW

**Review by TimPietrusky — APPROVED** [review]
LGTM

File: src/extra/expression/index.js [hunk 1]
```diff
@@ -11,6 +11,31 @@ const Expression = {
   store: expressionStore,
   storeName: 'expression',
 
+  presetData: {
```
Comment (TimPietrusky) [inline src/extra/expression/index.js:14]: You changed the path to `/@store` in another PR, but you didn't update the path in this file, only in `src/extra/expression/store.js`. But in `src/extra/expression/store.js`, the import to store is removed completely. Might this be a problem?

**Review by TimPietrusky — CHANGES_REQUESTED** [review]
@2xAA I have a question regarding loading the `store`

File: src/extra/expression/index.js (hunk 1 above)
Comment (TimPietrusky) [reply src/extra/expression/index.js:14]: But maybe I'm just wrong and this will be resolved by the merge anyway.

Comment (2xAA) [reply src/extra/expression/index.js:14]: I wouldn't be able to change the store location before the merge of the store PR. I was hoping Git would take care of that or I'd have to rebase some PRs.

Just wanted to get some work done while I had the chance.

**Pull request merged**